src/main/java/com/epam/esm/util/PropertyReader.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.esm.util;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyReader {
+    private static final Properties properties = new Properties();
+    private static PropertyReader instance = null;
+
+    public static Properties getProperties() {
+        if (instance == null) {
+            loadProperties();
+            instance = new PropertyReader();
+        }
+        return properties;
+    }
+    /**
+     * Reading properties from application.properties
+     */
+    public static void loadProperties() {
+        String propertiesFileName = "/Users/andrew_wannasesh/folders/" +
+                "EPAMLAB/EPAM_LAB/src/main/resources/application.properties";
+        InputStream inputStream = null;
+        try {
+            inputStream = new FileInputStream(propertiesFileName);
+            properties.load(inputStream);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
This method is supposed to read application properties, so you should shut it down here instead of printing stacktrace. The same applies to the catch in finally block. printStackTrace is never an option for exception handling, you should either take care of the exceptional situation or throw a new exception.